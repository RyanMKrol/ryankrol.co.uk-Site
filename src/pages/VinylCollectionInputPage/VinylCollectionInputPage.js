import React from 'react';

import { VinylCollectionForm } from '../../components';

import styles from './VinylCollectionInputPage.module.css';

/**
 * Page introducing the site
 *
 * @returns {string} JSX Content
 */
function VinylCollectionInputPage() {
  return (
    <div className={styles.container}>
      <h1 className="light">Add to the collection!</h1>
      <VinylCollectionForm />
    </div>
  );
}

export default VinylCollectionInputPage;
