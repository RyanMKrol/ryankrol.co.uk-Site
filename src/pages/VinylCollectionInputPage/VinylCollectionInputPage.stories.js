import React from 'react';

import VinylCollectionInputPage from './VinylCollectionInputPage';

export default {
  title: 'Pages/VinylCollectionInputPage',
};

export const Default = () => <VinylCollectionInputPage />;
