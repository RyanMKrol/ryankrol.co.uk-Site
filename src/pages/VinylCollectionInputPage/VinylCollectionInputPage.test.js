import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import VinylCollectionInputPage from './VinylCollectionInputPage';

it('renders VinylCollectionInputPage correctly', () => {
  const header = shallow(<VinylCollectionInputPage />);
  expect(toJson(header)).toMatchSnapshot();
});
