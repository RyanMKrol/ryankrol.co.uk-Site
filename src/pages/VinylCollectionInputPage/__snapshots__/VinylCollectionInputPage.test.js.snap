// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders VinylCollectionInputPage correctly 1`] = `
<div
  className="container"
>
  <h1
    className="light"
  >
    Add to the collection!
  </h1>
  <VinylCollectionForm />
</div>
`;
